src/vivarium_cluster_tools/psimulate/branches.py

```diff
@@ -129,16 +129,20 @@ def calculate_input_draws(
         existing draw numbers.
 
     """
+    MAX_DRAW_COUNT = 1000
```

Comment:
There's a near-zero chance this would ever change at IHME, right? Otherwise is there a centralized globals.py or something similar this can be stored?

Reply:
I'm not aware of a central location where constants like this are defined. If this did change it would presumably be accompanied by some sort of notice, so we could change it with them. I suspect it would screw with a lot of things if that changed, but I don't think it's worth worrying about right now.